services/iam/config/routes.rb

```diff
@@ -3,7 +3,6 @@
 Ros::Iam::Engine.routes.draw do
   jsonapi_resources :public_keys
   devise_for :users, module: 'users', defaults: { format: :json }
-  devise_for :roots, module: 'roots', defaults: { format: :json }
```

Comment:
Why are we removing this?

Reply:
Because it overrides another route for `root` here:
https://github.com/rails-on-services/ros/blob/ff7995a000c2f644b97045bb1a4e1f0aa8cf5a9b/services/iam/config/routes.rb#L10